src/components/web-page/index.js

```diff
@@ -51,6 +51,13 @@ class WebPage extends React.Component {
         this.props.onUrl(newUrl);
       }
     });
+
+    // Capture link clicks on page and update state with new url
+    currentWebView.addEventListener('did-navigate', (event) => {
+      this.setState({
+        url: event.url
+      });
+    });
```

Comment:
I know that need to more code about update url.

**`did-navigate` event doesn't cover [SPA(Single Page Application)](https://en.wikipedia.org/wiki/Single-page_application) Page redirect like using `react-router`.**

So Must use `did-navigate-in-page` event.

Following sentence is [document of `did-navigate-in-page`](https://electronjs.org/docs/api/webview-tag#event-did-navigate-in-page)
> When in-page navigation happens, the page URL changes but does not cause navigation outside of the page. 

## Example 
If you enter [sample page](https://app.modusign.co.kr), you may be enter `https://app.modusign.co.kr` but webpage redirect you from `https://app.modusign.co.kr` to `https://app.modusign.co.kr/authentication/signin` . You can check in normal-browser like Chrome. If you use only `did-navigate` event, url doesn't change to `https://app.modusign.co.kr/authentication/signin`

## Recommend
I recommend to write more code without deleting current code.

```
    currentWebView.addEventListener('did-navigate-in-page', (event) => {
      this.setState({
        url: event.url
      });
    });
```

Reply:
Thank you 👍 Have added it to PR.